content/about/feature-stages/index.md

```diff
@@ -85,8 +74,8 @@ Below is our list of existing features and their current phases. This informatio
 
 | Feature           | Phase
 |-------------------|-------------------
-| [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/kubernetes/)        | Beta
-| [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Beta
+| [Kubernetes: Envoy Installation and Traffic Interception](/docs/setup/kubernetes/)        | Stable
+| [Kubernetes: Istio Control Plane Installation](/docs/setup/kubernetes/) | Stable
 | [Kubernetes: Istio Control Plane Upgrade](/docs/setup/kubernetes/) | Beta
 | [Pilot Integration into Kubernetes Service Discovery](/docs/setup/kubernetes/)         | Stable
```

Comment:
What exactly is this item (Pilot Integration into Kubernetes Service Discovery)?  The link seems to be wrong.

Reply:
It didn't make much sense to me. But this seemed to be there already

Reply:
@jasmine-jaksic maybe you know?  ^^^.

Reply:
I think this entire section needs to be cleaned up. Its got some content that does not make much sense anymore